ui/components/app/srp-input/srp-input.js

```diff
@@ -35,9 +35,11 @@ export default function SrpInput({ onChange, srpText }) {
   const onSrpChange = useCallback(
     (newDraftSrp) => {
       let newSrpError = '';
+      let emptyDraftSrp = true;
       const joinedDraftSrp = newDraftSrp.join(' ');
 
       if (newDraftSrp.some((word) => word !== '')) {
```

Comment:
`newDraftSrp.some((word) => word !== '')` represents the same thing as `emptyDraftSrp` does, I believe. 

So we could do:

```
const hasPopulated = newDraftSrp.some((word) => word !== '');

if(hasPopulated) {
```

We can avoid the extra `emptyDraftSrp` with this.